src/actions/constants.js

```diff
@@ -0,0 +1,8 @@
+const ACTION_TYPE = {
```

Comment:
We'll need to establish a convention for this filename. The Signup PR has `actionTypes.js`. We can agree on one and stick to it.

Reply:
@brucemakallan the `constants.js` file has been changed to `actionTypes.js` as recommended.